src/Microsoft.DncEng.SecretManager/Readme.md

```diff
@@ -0,0 +1,196 @@
+# The .NET Core Engineering Secret Manager
+This tool is used to track and rotate the secrets in one or more key vaults, and to validate the usages of the same. This is done through a yaml manifest file for each vault, and json settings files for each service.
+
+## Rotating secrets
+The synchronize command can be used to rotate secrets that need rotation. It will automatically rotate all secrets that have either expired or hit their rotation time. This command is run as part of a scheduled build to keep all secrets up to date. Any secrets that can be rotated without a human involved will be rotated automatically. For secrets that require human interaction, the build will fail and a human will have to run synchronize.
+
+## Creating new secrets
+New secrets can be created by adding new entries to the manifest file, then running synchronize. The tool will then create the new secret and add it to the secret store.
```

Comment:
> then running synchronize.

Does this mean there is a CLI component? If so, some documentation about its operation would be good. 

Is this what one would do to set up a new service? Consider having a walk-through section for whatever scenarios you expect to be common (like configuring a new service, migrating an existing service to this system).

Reply:
The whole thing is a CLI, any operation will be through a command line operation.